ms-docs/JBoss EAP on Azure.MD

```diff
@@ -72,46 +72,46 @@ Solution architecture for these templates includes:
 * JBoss EAP clustered using Azure Virtual Machine Scale Set
 
 #### Linux Workload Migration for JBoss EAP
-Azure Workload Builder will simplify the Proof-of-Concept (POC), evaluation and migration process for Java apps running JBoss EAP on premises to Azure. It integrates with Azure migrate discovery to identify JBoss EAP servers. Then it will dynamically generate Ansible playbook for JBoss EAP server deployment. It leverages the RH MTA tool to migrate servers from other app servers to JBoss EAP. Steps for simplifying migration includes:
-* JBoss EAP clustered using VMSS (Azure Virtual Machine Scale Set)	* Assessment - scans applications and infrastructure
-* Infrastructure configuration - creates workload profile
-* Deployment and testing - deploy, migrate and test workload
-* Post-deployment configuration - integrates with data, monitoring, security, backup, and more
+Azure Workload Builder will simplify the Proof-of-Concept (POC), evaluation, and migration process for on-premises Java apps to Azure. The Workload Builder integrates with Azure Migrate Discovery tool to identify JBoss EAP servers. Then it will dynamically generate Ansible playbook for JBoss EAP server deployment. It leverage the Red Hat MTA tool to migrate servers from other app servers to JBoss EAP. Steps for simplifying migration include:
+* Evaluation - JBoss EAP clusters using Azure VM or Virtual Machine Scale Set
+* Assessment - Scans applications and infrastructure
+* Infrastructure configuration - Creates workload profile
+* Deployment and testing - Deploy, migrate, and test workload
+* Post-deployment configuration - Integrates with data, monitoring, security, backup, and more
 
 ## Server configuration choice
 
-For deployment of RHEL VM, you can either choose between PAYG or BYOS. For VM image from the [Azure Marketplace](https://azuremarketplace.microsoft.com) this offer defaults to a PAYG image. Deploy a BYOS type RHEL VM if you have your own RHEL OS image. Make sure your RHSM account has BYOS entitlement via Red Hat Cloud Access before you deploy the VMs or Virtual Machine Scale Set.
+For deployment of RHEL VM, you can either choose between PAYG or BYOS. Images from the [Azure Marketplace](https://azuremarketplace.microsoft.com) defaults to PAYG. Deploy a BYOS type RHEL VM if you have your own RHEL OS image. Make sure your RHSM account has BYOS entitlement via F Cloud Access before you deploy the VMs or Virtual Machine Scale Set.

```

Comment:
There is a letter 'F' before the word Cloud Access which I think is a typo.

Reply:
> There is a letter 'F' before the word Cloud Access which I think is a typo.

Please fix. Also code blocks starting in the new year needs to be this format:
'''
code
'''

Reply:
Noted and made these changes